Sources/HAP/Endpoints/pairSetup().swift

```diff
@@ -56,6 +57,7 @@ func pairSetup(device: Device) -> Application {
             case .keyExchangeRequest:
                 let session = try getSession(connection)
                 response = try controller.keyExchangeRequest(data, session)
+                _ = device.onPairingEvent.map { $0(device, .pairingCompleted) }
```

Comment:
The response could also be an error at this point. We shouldn't hook into these functions, but do that on the actual `Device` instance. These callbacks should be fired from `Device.add(pairing:)` and `Device.remove(pairingWithIdentifier:)` instead.